packages/desktop-client/src/components/mobile/MobileNavTabs.tsx

```diff
@@ -179,23 +181,36 @@ export function MobileNavTabs() {
         backgroundColor: theme.mobileNavBackground,
         borderTop: `1px solid ${theme.menuBorder}`,
         ...styles.shadow,
-        height: totalHeight,
+        height: totalHeight + PILL_HEIGHT,
         width: '100%',
         position: 'fixed',
         zIndex: 100,
         bottom: 0,
         ...(!isNarrowWidth && { display: 'none' }),
       }}
     >
-      <View
-        style={{
-          flexDirection: 'row',
-          flexWrap: 'wrap',
-          height: totalHeight,
-          width: '100%',
-        }}
-      >
-        {[navTabs, bufferTabs]}
+      <View>
+        <div
+          style={{
+            background: theme.mobileHeaderTextSubdued,
```

Comment:
We should create a new color for this e.g. mobileNavDragHandle and define a lighter color for each theme we have (light, dark, midnight, developement)

Reply:
Changed it with `pillBorder`, if it will not suit your taste i will create the new color